project/plugins.sbt

```diff
@@ -13,3 +13,4 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0.BETA1")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.0.4")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.2")
+addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.6.0")
```

Comment:
does this need to be in the repo? Can't you put it in your personal `~/.sbt/` config?